src/sonic-yang-models/tests/yang_model_tests/tests_config/acl.json

```diff
@@ -1748,5 +1748,355 @@
                 ]
             }
         }
+    },
+    "ACL_TABLE_INNERSRCMACREWRITE_TABLE": {
+        "sonic-acl:sonic-acl": {
+            "sonic-acl:ACL_RULE": {
+                "ACL_RULE_LIST": [
+                    {
+                        "ACL_TABLE_NAME": "SRC-MAC-REWRITE-TABLE",
+                        "INNER_SRC_MAC_REWRITE_ACTION": "60:45:BD:08:19:68",
+                        "PRIORITY": 999960,
+                        "RULE_NAME": "Rule_10",
+                        "INNER_SRC_IP": "10.176.0.0/15",
+                        "VNI": 45
+                    },
+                    {
+                        "ACL_TABLE_NAME": "SRC-MAC-REWRITE-TABLE",
+                        "INNER_SRC_MAC_REWRITE_ACTION": "60:45:BD:08:19:68",
+                        "PRIORITY": 999960,
+                        "RULE_NAME": "Rule_9",
+                        "INNER_SRC_IP": "10.166.0.0/15",
+                        "VNI": 47
+                    }
+                ]
+            },
+            "sonic-acl:ACL_TABLE": {
+                "ACL_TABLE_LIST": [
+                    {
+                        "ACL_TABLE_NAME": "SRC-MAC-REWRITE-TABLE",
+                        "policy_desc":"Replace src MAC with ENI MAC",
+                        "ports": [
+                            "Ethernet0",
+                            "Ethernet1"
+                        ],
+                        "stage": "EGRESS",
+                        "type": "L2"
```

Comment:
in the ut can you create a new ACL_TABLE_TYPE configuration instead of using a predefined/sw created config called L2?